src/components/CookieMessage/_cookie-message.scss

```diff
@@ -0,0 +1,115 @@
+// TODO: conform styles to Wellcome Design System
```

Comment:
Presume these are copy-pasted from D7, with (spacing/colour) variables inserted?

Reply:
yes indeed. I've literally just added some blurb about that in the description :)